mappings/net/minecraft/client/gui/screen/ingame/BookScreen.mapping

```diff
@@ -1,18 +1,24 @@
 CLASS net/minecraft/class_3872 net/minecraft/client/gui/screen/ingame/BookScreen
 	CLASS class_3931 Contents
-		METHOD method_17560 getLineCount ()I
-		METHOD method_17561 getLine (I)Lnet/minecraft/class_2561;
-			ARG 1 line
+		METHOD method_17560 getPageCount ()I
+		METHOD method_17561 getPage (I)Lnet/minecraft/class_2561;
```

Comment:
`getPageUnchecked`

Reply:
Does unchecked imply a crash if invalid? Or null return?

Reply:
unchecked means like getting something unsafe, like nullable

Reply:
The null value isn't inherently unsafe -- only when it is dereferenced.

Reply:
the unchecked method here may throw index out of bounds exception and is impl only; all calls actually go through `getPageOrDefault`, which imo should be renamed to `getPage`